### DISCUSSION TITLE
will it support multiple data bases

### DISCUSSION THREAD

**Discussion by Bajibabup** [sidorares/node-mysql2#1905] [General]
Hi,
I read that mysql2 supports MariaDB, Oracle DB also. Pls can you confirm if it supports.

**Comment by sidorares** [comment]
The goal of mysql2 is to be a client for mysql protocol - https://dev.mysql.com/doc/dev/mysql-server/latest/PAGE_PROTOCOL.html

I know people are using this driver with various databases that talk mysql protocol: MariaDB, Percona DB, [Manticore Search](https://manticoresearch.com/), [Sphinx Search](http://sphinxsearch.com/), PlanetScale

Currently we have mysql, mariadb and PlanetScale in the CI matrix.

I'm open to supporting extensions to the protocol that other DBs may have, but completely different protocol is probably outside of the scope for this client.
